rules/CodingStyle/Rector/Enum_/EnumCaseToPascalCaseRector.php

```diff
@@ -187,7 +187,28 @@ private function refactorClassConstFetch(ClassConstFetch $classConstFetch): ?Nod
 
     private function convertToPascalCase(string $name): string
     {
+        if ($this->isPascalCase($name)) {
```

Comment:
use existing `convertToPascalCase()` method, compare, if same, then skip

Reply:
That's not going to work here, the `convertToPascalCase()` method is not idempotent

Reply:
that strange that the check already there

https://github.com/rectorphp/rector-src/blob/87d442710e116b0141ed8961a3f1b56311785534/rules/CodingStyle/Rector/Enum_/EnumCaseToPascalCaseRector.php#L104-L107

I am on mobile, I will check when I am back to PC